src/routes/admin/wireless/wirelessValidator.ts

```diff
@@ -0,0 +1,104 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2019
+ * SPDX-License-Identifier: Apache-2.0
+ * Author : Madhavi Losetty
+ **********************************************************************/
+import { check } from 'express-validator'
+
+const isValidLinkPolicy = (n: number): boolean => {
+  // ValueMap={1, 14, 16, 224}
+  // Values={available on S0 AC, available on Sx AC, available on S0 DC, available on Sx DC}
+  return (n === 1 || n === 14 || n === 16 || n === 224)
+}
+
+export const wirelessValidator = (): any => {
+  return [
+    check('profileName')
+      .not()
+      .isEmpty()
+      .withMessage('Wireless profile name is required')
+      .matches('^[a-zA-Z0-9]+$')
+      .isLength({ max: 32 })
+      .withMessage('Wireless profile name should be alphanumeric and maximum length is 32'),
+    check('authenticationMethod')
+      .not()
+      .isEmpty()
+      .withMessage('Authentication Method is required')
+      .isIn([4, 6])
+      .withMessage('Authentication method should be either 4(WPA PSK) or 6(WPA2 PSK)'),
+    check('encryptionMethod')
+      .not()
+      .isEmpty()
+      .withMessage('EncryptionMethod Method is required')
```

Comment:
method should not be captialized and EncryptionMethod should be two words, remove second method

Reply:
done